src/services/refactors/extractSymbol.ts

```diff
@@ -5,7 +5,7 @@
 namespace ts.refactor.extractSymbol {
     const extractSymbol: Refactor = {
         name: "Extract Symbol",
-        description: Diagnostics.Extract_symbol.message,
+        description: getLocaleSpecificMessage(Diagnostics.Extract_symbol),

```

Comment:
nit. i think we should call it `extract local` instead of `extract symbol`

Reply:
As we discussed offline, that description goes on the `ApplicableRefactorInfo`.